docs/examples/block_ip.c

```diff
@@ -29,6 +29,11 @@
  * filter IP addresses.
  */
 
+#ifdef __AMIGA__
+#include <stdio.h>
+int main(void) { printf("amigaos isn't supported.\n"); return 1; }
```

Comment:
```suggestion
int main(void) { printf("AmigaOS is not supported.\n"); return 1; }
```
Tiny nit to sync casing with other mentions (and drop the contraction while there).